cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -113,6 +113,7 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 	}
 
 	var r []error
+	var targetResults []*target.TargetResult
```

Comment:
My suggestion would be to merge the `r` with `targetResults`, so that any errors relevant to the target are part of its result. This will in my opinion make it easier to connect errors and target-specific results on the caller side, once we would allow more than one target to be specified.